src/renderer/pkijs/generatePems/common.ts

```diff
@@ -0,0 +1,85 @@
+import { fromBER } from 'asn1js'
+import { stringToArrayBuffer, fromBase64 } from 'pvutils'
+import fs from 'fs-sync'
+import {
+  getAlgorithmParameters, getCrypto, setEngine,
+  CryptoEngine, Certificate, CertificationRequest,
+  SignedData
+} from 'pkijs'
+
+const webcrypto = new (require('node-webcrypto-ossl')).Crypto()
```

Comment:
you can't do this on module level.
this will make it impossible to import in the browser, because `node-webcrypto-ossl` is only for node.js envrionment (which is what you used for your tests)